CMakeLists.txt

```diff
@@ -79,11 +79,14 @@ target_link_libraries(multiprocess PRIVATE CapnProto::kj)
 target_link_libraries(multiprocess PRIVATE CapnProto::kj-async)
 set_target_properties(multiprocess PROPERTIES
     PUBLIC_HEADER "${MP_PUBLIC_HEADERS}")
-install(TARGETS multiprocess EXPORT Multiprocess
+install(TARGETS multiprocess EXPORT libmultiprocess-lib
   ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT lib
   PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/mp COMPONENT lib)
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/pkgconfig/libmultiprocess.pc"
   DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig COMPONENT lib)
+install(EXPORT libmultiprocess-lib
+  NAMESPACE libmultiprocess::
```

Comment:
In commit "Install Exports for custom `install-{lib,bin}` targets" (dae3ff8827e4803f601c764ac4664815b14251b9)

Could you change `NAMESPACE libmultiprocess::` to `NAMESPACE Libmultiprocess::` here and below. Seems better not to use different namespace and project names unnecessarily, and from https://cmake.org/cmake/help/latest/guide/importing-exporting/index.html, it seems like the convention for both is to use uppercase